tests/NLog.UnitTests/Config/ConfigApiTests.cs

```diff
@@ -135,6 +134,35 @@ public void AddRule_min_max()
             Assert.False(rule1.IsLoggingEnabledForLevel(LogLevel.Off));
         }
 
+        [Fact]
+        public void AddRule_ruleobject()
+        {
+            var config = new LoggingConfiguration();
+            config.AddTarget(new FileTarget { Name = "File" });
+            LoggingRule rule = new LoggingRule("testRule")
+            {
+                LoggerNamePattern = "testRulePattern"
+            };
+            rule.EnableLoggingForLevels(LogLevel.Info, LogLevel.Error);
+            rule.Targets.Add(config.FindTargetByName("File"));
+            rule.Final = true;
+            config.AddRule(rule);
+            Assert.NotNull(config.LoggingRules);
+            Assert.Equal(1, config.LoggingRules.Count);
+            var rule1 = config.LoggingRules.FirstOrDefault();
```

Comment:
Since it is `config.AddRule(rule);` could you use replace `FirstOrDefault()` with `LastOrDefault()` ?